skel/daemons/settings.go

```diff
@@ -0,0 +1,368 @@
+package daemons
```

Comment:
I think we need to align the name of the file and its function names
Maybe it should be something like `configuration.go`?